app/src/main/java/org/simple/clinic/facility/FacilityRepository.kt

```diff
@@ -18,7 +18,7 @@ class FacilityRepository @Inject constructor(
     val userFacilityMappingDao: LoggedInUserFacilityMapping.RoomDao
 ) {
 
-  fun facilities(): Observable<List<Facility>> {
+  fun facilitiesInAscending(): Observable<List<Facility>> {
```

Comment:
Accepting better names. I also considered,

- Adding a boolean param `facilities(orderByAscending=true)`
- Adding an enum `facilities(order = ASC)`

Reply:
Naming is tricky. This name doesn't indicate what to order by ascending; it could be by the name, or the date at which it was created.

I would suggest just changing the name to `facilitiesSortedByName()`, which implies a natural ordering and leave it for now. We can revisit this if the UI needs it in future (with user selected sorting order or something similar).

Reply:
IMHO, we should not change this to anything. Let it remain `facilities()` -- we don't need to expose the sorting order at all, because we don't care about the order (or about changing it) right now.